File: php/testcode/benchmark_test_00129.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00129(BenchmarkRequest $req): BenchmarkResponse {
    $tplName  = $req->param('tpl');
    $cacheFile = CACHE_DIR . sha1($tplName) . '.php';
    include $cacheFile;
    return BenchmarkResponse::ok('Cached template rendered');
}

File: rust/testcode/benchmark_test_00129.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let preview_url = req.param("preview_url");

    let html = format!(
        "<html><body><iframe src='{}' width='600' height='400'></iframe></body></html>",
        preview_url
    );

    super::shared::BenchmarkResponse::ok(&html)
}
